CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing Guidelines
+
+## Checklist
+
+* Make sure your PR follows the [coding guidelines](https://github.com/ggerganov/llama.cpp/blob/master/README.md#coding-guidelines)
+* Test your changes using the commands in the [`tests`](tests) folder. For instance, running the `./tests/test-backend-ops` command tests different backend implementations of the GGML library
+* Execute [the full CI locally on your machine](ci/README.md) before publishing
+
+## PR formatting
+
+* Please rate the complexity of your PR (i.e. `easy`, `medium`, `hard`). This makes it easier for maintainers to triage the PRs.
```

Comment:
This should probably reflect the label names

```suggestion
* Please rate the complexity of your PR (i.e. `low`, `medium`, `high`). This makes it easier for maintainers to triage the PRs.
```

Reply:
Agreed. I do wonder if it's reasonable to ask people to type something like `medium complexity` as that might be less ambiguous to match against a regex in the future.

Reply:
directly edited it just now so all good